theme/component-selector.html

```diff
@@ -1,7 +1,7 @@
 <div class="component-selector">
     <h4 class="component-selector__title">Components</h4>
     <h5 class="component-selector__subtitle">Find documentation…</h5>
-    <select class="form-control component-selector__control">
+    <select id="component-selector-control" class="form-control component-selector__control">
```

Comment:
Is the `id` attribute really needed? I can find no reference.

Reply:
Nope, not needed; I thought I'd omitted it in my most recent changes. I'll revert it.